variables.robot

```diff
@@ -221,6 +221,10 @@ ${OS_UBUNTU}=               ubuntu
 # NovaCustom V560TU -----------------------------------------------------------
 &{RTE62}=                   ip=0.0.0.0
 ...                         platform=novacustom-v560tu    platform_vendor=Clevo
+# Protectli V1211
+&{RTE63}=                   ip=192.168.10.227    cpuid=02c00042652f4120
```

Comment:
no need to add cpuid anymore, see how most of the setups does not have it

Reply:
It was added by Dawid Lesiak. I didn't remove it as it was already there.